### PR TITLE
Update es_rakuten.m3u

### PR DESCRIPTION
**Pull request by freearhey**
- updated links to streams (https://github.com/iptv-org/iptv/pull/7128#issuecomment-1106731351)
- broken links removed
- created `de_rakuten.m3u`, `fr_rakuten.m3u`, `it_rakuten.m3u`, `nl_rakuten.m3u`, `uk_rakuten.m3u` (https://github.com/iptv-org/iptv/pull/7271#discussion_r870519105)

### CODE REVIEW

File: streams/es_rakuten.m3u [hunk 1]
```diff
@@ -95,41 +83,41 @@ https://rakuten-topfree-3-eu.rakuten.wurl.tv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 1 (720p)
+https://rakuten-tvshows-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 2 (720p)
+https://rakuten-tvshows-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 3 (720p)
+https://rakuten-tvshows-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 6 (720p)
+https://rakuten-tvshows-6-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 7 (720p)
+https://rakuten-tvshows-7-eu.rakuten.wurl.tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/es_rakuten.m3u:121]: Anything having -7 on the URL seems to be French. Maybe adding it with the draft "fr_rakuten.m3u" link would be better ?

File: streams/es_rakuten.m3u [hunk 2]
```diff
@@ -95,41 +83,41 @@ https://rakuten-topfree-3-eu.rakuten.wurl.tv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 7 (720p)
+https://rakuten-spotlight-7-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 1 (720p)
+https://rakuten-tvshows-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 2 (720p)
+https://rakuten-tvshows-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 3 (720p)
+https://rakuten-tvshows-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 6 (720p)
+https://rakuten-tvshows-6-eu.rakuten.wurl.tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/es_rakuten.m3u:119]: Anything having -6 on the URL seems to be Italian. Maybe splitting the links should be better ?

Comment (LaneSh4d0w) [reply streams/es_rakuten.m3u:119]: and the -5 are German.

File: streams/es_rakuten.m3u [hunk 3]
```diff
@@ -95,41 +83,41 @@ https://rakuten-topfree-3-eu.rakuten.wurl.tv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 6 (720p)
+https://rakuten-spotlight-6-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 7 (720p)
+https://rakuten-spotlight-7-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 1 (720p)
+https://rakuten-tvshows-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 2 (720p)
+https://rakuten-tvshows-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 3 (720p)
+https://rakuten-tvshows-3-eu.rakuten.wurl.tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/es_rakuten.m3u:117]: -3 are also German ???

File: streams/es_rakuten.m3u [hunk 4]
```diff
@@ -95,41 +83,41 @@ https://rakuten-topfree-3-eu.rakuten.wurl.tv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 3 (720p)
+https://rakuten-spotlight-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 6 (720p)
+https://rakuten-spotlight-6-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 7 (720p)
+https://rakuten-spotlight-7-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 1 (720p)
+https://rakuten-tvshows-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 2 (720p)
+https://rakuten-tvshows-2-eu.rakuten.wurl.tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/es_rakuten.m3u:115]: Those are Spanish.

File: streams/es_rakuten.m3u [hunk 5]
```diff
@@ -95,41 +83,41 @@ https://rakuten-topfree-3-eu.rakuten.wurl.tv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 2 (720p)
+https://rakuten-spotlight-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 3 (720p)
+https://rakuten-spotlight-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 6 (720p)
+https://rakuten-spotlight-6-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 7 (720p)
+https://rakuten-spotlight-7-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 1 (720p)
+https://rakuten-tvshows-1-eu.rakuten.wurl.tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/es_rakuten.m3u:113]: -1 are English.

File: streams/nl_rakuten.m3u [hunk 6]
```diff
@@ -0,0 +1,7 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies Netherlands (720p)
+https://rakuten-comedymovies-5-eu.rakuten.wurl.tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/nl_rakuten.m3u:3]: It actually is German, sorry for confusing both languages. Felt weird that Germany had 2 slots.

**Comment by LaneSh4d0w** [comment]
Fusioned my PR with yours.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Central302 — APPROVED** [review]

**Pull request merged**